open_alaqs/core/modules/MovementSourceModule.py

```diff
@@ -143,7 +143,7 @@ def addAdditionalColumnsToDataFrame(self):
         # Create a function that returns a list of default emissions
         def _default_emissions(*args):
             return {
-                "emissions": default_emission,
+                "emissions": [default_emission],
```

Comment:
This may have been the real source for the issue (I dare to remove and fix a TODO there), since the Movement.py modifications didn't alter return types.